### ISSUE TITLE
迷ったときに、参考になる・聞ける場所はありますか

### ISSUE THREAD

**Issue by tokimari** [insidefrontend/issue2-ama#38]
# 質問

いざ社内でやっていこうというときに、方針がわからなかったり迷ったときに、参考になる・聞ける場所はありますか

# 質問をしたいAMAブース

- [ ] Web Payments / Payment Request API について何でも訊いて下さい by えーじ
- [ ] AbemaTVだけど質問ある？（AbemaTV関係ない質問でもいいです）by Kenichi Kato, Tetsuharu OHZEKI, and Shota Kubota
- [ ] 日経電子版の高速化について何でも聞いてください by sisidovski and cssradar
- [ ] Fastly のエッジクラウドについて何でも聞いてください by Fastly
- [ ] Web パフォーマンスについて何でも訊いて下さい by Ayumu Sato and Shogo Sensui
- [x] freeeのアクセシビリティ、いまとこれから by 伊原 力也
- [ ] 現場の ES201x とアーキテクチャの変遷と未来 by mizchi
- [ ] デザインとエンジニアリングのワークフローについて語りましょう by Hidemi Atarashi, Kensuke Takada, Masanari Hamada
- [ ] コンポーネント座談会 by pirosikick, 伊藤康太, 加藤佑典

**Comment by tokimari** [comment]
# 回答

**伊原さん：** a11yjというslackチームがあるので、そこで聞くと結構返ってくるので、ぜひ参加してみてください。

アクセシビリティSlack Teamはここから入れます。
https://a11yj.herokuapp.com/
